mapper/thin_edge_json/src/serialize.rs

```diff
@@ -154,13 +136,19 @@ impl GroupedMeasurementVisitor for ThinEdgeJsonSerializer {
             return Err(MeasurementStreamError::UnexpectedEndOfGroup.into());
         }
 
-        self.buffer.push('}');
+        self.json.write_close_obj();
         self.needs_separator = true;
         self.is_within_group = false;
         Ok(())
     }
 }
 
+impl ThinEdgeJsonSerializer {
+    pub fn into_string(&mut self) -> Result<String, ThinEdgeJsonSerializationError> {
+        self.end()?;
+        Ok(self.json.clone().into_string())
+    }
+}
```

Comment:
I prefer to have a *single* block to define the methods of `ThinEdgeJsonSerializer`.

* This is easier to read.
* This might suggest code re-use between the `into_string` and `bytes`  methdos.

Reply:
Yes, I agree. I somehow missed it. I have corrected it.